css/styles.css

```diff
@@ -58,15 +60,18 @@
     margin: 0;
     font-size: 14px;
     font-weight: normal;
+    line-height: 20px;
     text-transform: uppercase;
 }
 .ab-column h3.icon:before {
-    font: normal 28px/28px icons;
+    font: normal 30px/30px icons;
     top: 4px;
-    right: 0;
+    right: 20px;
```

Comment:
I guess this change was necessary due to some other (maybe `ab-column` margin?) changes. With the original CSS as of 2d193ccc36be60025bd5d8d82f6238b0203b5ffb `right: 0;` results in misaligned icons, relative to their respective columns.
![bildschirmfoto 2017-02-05 um 21 55 54](https://cloud.githubusercontent.com/assets/1038789/22629821/e906e69e-ebed-11e6-99ba-e55e728ebb10.png)